credentialdigger/snippet_similarity.py

```diff
@@ -0,0 +1,38 @@
+import tensorflow as tf
+import tensorflow_text
+import logging
+import tensorflow_hub as hub
+import tensorflow.keras.preprocessing.text
+import numpy as np

```

Comment:
The order of the imports is wrong (you can use [`isort`](https://pycqa.github.io/isort/) to do it automatically)